content/en/developers/dogstatsd/_index.md

```diff
@@ -64,10 +65,54 @@ By default, DogStatsD listens on UDP port **8125**. If you need to change this,
 
 2. [Restart your Agent][10].
 
+#### DogStatsD and Docker
```

Comment:
We should use a tab logic instead of a full section.

Reply:
Since K8s DogStatsD instructions are already written, we could add another tab "Kubernetes" that links to the specific page.